### PR TITLE
Palo alto network operations pack

### PR DESCRIPTION
**Pull request by adambaumeister**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress

related to: https://jira-hq.paloaltonetworks.local/browse/CIAC-677

## Description
@GuyAfik @tomer-pan 
This is the entire content pack that comprises PAN-OS Network operations. It includes the following use cases;

 - Firewall/Panorama Upgrades
 - System Snapshotting and comparisons
 - Configuration Hygiene Checks

This pack includes a large set of supporting scripts, besides the playbooks, types, fields, and layouts. 

This content was exported using the marketplace "contribute" button and then converted into the preferred directory structure. Because of this there are likely UUID and other such issues present, but it should still be a valid content pack either way.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [X] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/PAN_OS_Network_Operations/Scripts/CountActiveVisibilityHygieneIssues/CountActiveVisibilityHygieneIssues.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        }
+    }
+}
+
+# If demisto.args() isn't empty, then this is being invoked as a widget script, in which case simply return the count.
+# Also check for currentUser key, which deliniates this as being run within an incident layout
+if "currentUser" in demisto.args():
+    demisto.results(data)
+elif demisto.args():
+    demisto.results(count)
```
Comment (MosheEichler) [inline Packs/PAN_OS_Network_Operations/Scripts/CountActiveVisibilityHygieneIssues/CountActiveVisibilityHygieneIssues.py:49]:
```suggestion
if "currentUser" not in demisto.args():
    demisto.results(count)
```

File: Packs/PAN_OS_Network_Operations/Scripts/CountActiveVisibilityHygieneIssues/CountActiveVisibilityHygieneIssues.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                    "#FF9000": {"value": 0},
+                    "#FF1744": {"value": 10},
+                }
+            },
+            "type": "above"
+        }
+    }
+}
+
+# If demisto.args() isn't empty, then this is being invoked as a widget script, in which case simply return the count.
```
Comment (MosheEichler) [inline Packs/PAN_OS_Network_Operations/Scripts/CountActiveVisibilityHygieneIssues/CountActiveVisibilityHygieneIssues.py:44]: > If demisto.args() isn't empty
Is this not always empty? from the YML file it looks there are no args.

File: Packs/PAN_OS_Network_Operations/Scripts/CountFixedVisibilityHygieneIssues/CountFixedVisibilityHygieneIssues.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        }
+    }
+}
+
+# If demisto.args() isn't empty, then this is being invoked as a widget script, in which case simply return the count.
+# Also check for currentUser key, which deliniates this as being run within an incident layout
+if "currentUser" in demisto.args():
+    demisto.results(data)
+elif demisto.args():
+    demisto.results(count)
```
Comment (MosheEichler) [inline Packs/PAN_OS_Network_Operations/Scripts/CountFixedVisibilityHygieneIssues/CountFixedVisibilityHygieneIssues.py:47]: See the previous comments.

**Review by MosheEichler — CHANGES_REQUESTED** [review]
Hi @PierrickV.
You did an Excellent Job!! :)

Please add unit tests for all scripts.
And please see the above comments.
And let me know when you are done.

Thank you.

**Comment by adambaumeister** [comment]
Heya Team,

Gonna close this one as I'm going to split this into a couple of different packs instead.

**Pull request closed without merging**